phpBB/docs/events.md

```diff
@@ -1072,6 +1072,20 @@ navbar_header_logged_out_content
 * Since: 3.1.0-RC1
 * Purpose: Add text and HTML in place of the username when not logged in.
 
+navbar_header_user_profile_append
```

Comment:
I'm sorry but the placement is still wrong. In alphabetic order, navbar_header_u... should be after navbar_header_p...